systematics/genptshape/config_weights_nonPromptDs_PbPb010.yml

```diff
@@ -0,0 +1,38 @@
+meson:
+    Dspecie: Ds
+    Bspecie: BsBmix # options: ['Bs', 'Bplus', 'Bzero', 'Lb', 'BsBmix']
+
+inputMC:
+    filename: ptweights/DsPbPb/GenPtShape_LHC20g11a.root
```

Comment:
Why the Ds dedicated production is used instead of the D2H one (as done for the prompt)?

Reply:
There is no difference between the two since the MC config is exactly the same, just that we have more signal in the Ds dedicated one so I used that one